diff --git a/_cjs/internal/constants.js b/_cjs/internal/constants.js
index 14b006d13672f120c57f85a38b9a1bc02736e80c..d56372f5bce3619689cef1eaf154cca966880b57 100644
--- a/_cjs/internal/constants.js
+++ b/_cjs/internal/constants.js
@@ -40,7 +40,7 @@ const constants = {
         name: "@fhevm/host-contracts",
     },
     ZAMA_FHE_RELAYER_SDK_PACKAGE: {
-        version: "0.3.0-8",
+        version: "0.4.0-2",
         name: "@zama-fhe/relayer-sdk",
         sepolia: {
             relayerUrl: "https://relayer.testnet.zama.org",
